installer-guide/mac-install.md

```diff
@@ -5,73 +5,40 @@
 
 While you don't need a fresh install of macOS to use OpenCore, some users prefer having a fresh slate with their boot manager upgrades.
 
-To start we'll want to grab ourselves a copy of macOS. You can skip this and head to formatting the USB if you're just making a bootable OpenCore stick and not an installer. For everyone else, you can either download macOS from the App Store or with gibMacOS.
+To start we'll want to grab ourselves a copy of macOS. You can skip this and head to formatting the USB if you're just making a bootable OpenCore stick and not an installer. For everyone else, you can either download macOS from the App Store or with Munki's script.
 
 ## Downloading macOS: Modern OS
 
 * This method allows you to download macOS 10.13 and newer, for 10.12 and older see [Downloading macOS: Legacy OS](#downloading-macos-legacy-os)
 
 From a macOS machine that meets the requirements of the OS version you want to install, go directly to the App Store and download the desired OS release and continue to [**Setting up the installer**](#setting-up-the-installer).
 
-For machines that need a specific OS release or can't download from the App Store, you can use the gibMacOS utility.
+For machines that need a specific OS release or can't download from the App Store, you can use the Munki's InstallInstallMacOS utility.
 
-Now let's grab [gibMacOS](https://github.com/corpnewt/gibMacOS), then unzip it into a local directory.
+In order to run it, just copy and paste the below command in a terminal window:
 
-Next run the `gibMacOS.command`:
+`curl -O https://raw.githubusercontent.com/munki/macadmin-scripts/main/installinstallmacos.py && sudo /usr/bin/python installinstallmacos.py`
```

Comment:
Few things:

* Better to use the global `python` command for up-to-date binary, as  `/usr/bin/python` won't always have the latest
  * more a nitpick as I see the script is both py2 and py3 compliant anyways
* Is `sudo` needed for this? Not a fan of giving elevated privileges unless required
* Another recommendation would be to make a new folder and run there, so no need to dirty the user folder(ie. `mkdir macOS-Installers`)
  * Mostly nitpick again, not that important
* Incorrect usage of code block, please use triple backquote's instead: ```

Reply:
Tested, yeah `sudo`'s needed